samples/StaticHtmlComposites/src/callComposite.js

```diff
@@ -3,11 +3,17 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { AzureCommunicationTokenCredential } from '@azure/communication-common';
 import { CallComposite, createAzureCommunicationCallAdapter } from '@azure/communication-react';
 
 export const loadCallComposite = async function (args, htmlElement) {
   const { userId, token, groupId, displayName } = args;
-  const adapter = await createAzureCommunicationCallAdapter(userId, token, { groupId }, displayName ?? 'anonymous');
+  const adapter = await createAzureCommunicationCallAdapter(
```

Comment:
Was there a good reason for moving around the parameters in this function? I feel this is a breaking change? I am fine with a breaking change but do we have a good reason for this?

Reply:
These params should definitely be a named object too so order doesn't not matter (there's too many of them), maybe too late to do for this beta release though

Reply:
PR here if we do, will do chat also next, can then decide to cp to this beta or not https://github.com/Azure/communication-ui-library/pull/625